### PR TITLE
Unskip the LB over limit test (CATC 19)

### PR DESCRIPTION
**Pull request by cyli**
This is dependent upon #1512 getting merged too, which fixes a bug in otter where it just retries.

Our CI mimic version has been upgraded to include @kivattik's feature to supports CLB limits.

There were some changes to the assertions though, since it's hard to predict exactly how many servers will become active by the time we get the CLB error.  (otter errors on the first error from CLB, rather than attempt to give you the max it can and then errors).

So there should be at least the original servers that are active... and then at some point the group will go into ERROR.  There will definitely be less than the desired max though.

### CODE REVIEW

**Comment by cyli** [comment]
jenkins please run integration tests

File: otter/integration/tests/test_convergence.py [hunk 1]
```diff
@@ -1121,13 +1129,14 @@ def test_scale_over_lb_limit(self):
...
-                        'desiredCapacity': Equals(group_max),
-                        'status': Equals("ERROR")
-                    }),
-                    HasActive(LB_max)),
+                ContainsDict({
+                    'pendingCapacity': GreaterThan(group_max - LB_max - 1),
+                    'desiredCapacity': Equals(group_max),
+                    'status': Equals("ERROR"),
+                    'active': AfterPreprocessing(
+                        len, MatchesAll(GreaterThan(LB_max - 6),
```
Comment (kivattik) [inline otter/integration/tests/test_convergence.py]: I'm thinking this LB_max - 6 should either have a comment or reference an initial_offset variable to link this to the starting desired capacity in order to improve maintainability.

**Comment by kivattik** [comment]
:+1: After the maintainability comment. :)

**Comment by cyli** [comment]
jenkins please run integration tests

**Pull request merged**